src/CachePoolTest.php

```diff
@@ -674,7 +702,8 @@ public function testDataTypeArray()
         $item = $this->cache->getItem('key');
         $this->assertTrue(is_array($item->get()), 'Wrong data type. If we store array we must get an array back.');
         $this->assertEquals($array, $item->get());
-        $this->assertTrue($item->isHit(), 'Null is a perfectly fine cache value. isHit() should return true when null are stored. ');
+        $this->assertTrue($item->isHit(),
+            'Null is a perfectly fine cache value. isHit() should return true when null are stored. ');
```

Comment:
You sould avoid doing formatting changes when submitting a PR. It makes code review harder by adding lots of noise (at least half of the diff of this PR is irrelevant)